app/services/remote_job_handler.rb

```diff
@@ -83,44 +123,60 @@ def prepare_input_json(job)
     end
   end
 
-  def prepare_input_files(job)
-    return if job.is_a?(Run)
+  def prepare_input_files_for_analysis(job)
+    org_dest_list = job.input_files.map do |origin,dest|
+      [origin, Pathname('_input').join(dest)]
+    end
+    if @host.mounted_work_base_dir.present?
+      copy_files_to_work_dir_via_copy(job, org_dest_list)
+    else
+      copy_files_to_work_dir_via_ssh(job, org_dest_list)
+    end
+  end
+
+  def copy_results_of_local_pre_process(job)
+    return unless job.executable.local_pre_process_script.present?
+    org_dest_list = filelist_of_local_preprocess(job)
     if @host.mounted_work_base_dir.present?
-      prepare_input_files_via_copy(job)
+      copy_files_to_work_dir_via_copy(job, org_dest_list)
     else
-      prepare_input_files_via_ssh(job)
+      copy_files_to_work_dir_via_ssh(job, org_dest_list)
     end
   end
 
-  def prepare_input_files_via_copy(job)
-    remote_path = RemoteFilePath.input_files_dir_path(@host,job)
+  def filelist_of_local_preprocess(job)
+    org_dest_list = []
+    Dir.chdir(job.dir) {
+      org_dest_list = Dir.glob("**/*").map {|f| [ Pathname.new(f).expand_path, f ] }
```

Comment:
This is an intensional behavior as well. All the output files of a local pre-process are copied to the remote work directory. If you do not want to copy a certain file, you may remove it in the local pre-process.